packages/circuit-ui/components/Avatar/Avatar.docs.mdx

```diff
@@ -0,0 +1,46 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+
+# Avatar
+
+<Status.Stable />
+
+The Avatar component displays an identity or an object image. It can be passed to the [ImageInput](Forms/ImageInput) to allow users to upload an avatar.
+
+<Story id="components-avatar--base" />
+
+<Props />
+
+## Usage guidelines
+
+- **Do** use the right variant for your use case (see _Variants_ below).
+- **Do** use the [ImageInput component](Forms/ImageInput) with the `component={Avatar}` prop to allow users to upload an avatar (note: the ImageInput only supports `variant="object"` for now).
```

Comment:
What's the reason for the `variant="object"` limitation?

Reply:
(same as [above](https://github.com/sumup-oss/circuit-ui/pull/922#discussion_r643464874))

Reply:
Hm, that's annoying. I'll try to think of a solution 🤔

Reply:
It's also not mandatory for v3, it can be improved as a minor 🙂

Reply:
Let's keep it like this for now and think of a solution when we need it. As you said, this can be changed in a minor.